### PR TITLE
Bugfix/issue 16 keyword fanout

### PR DESCRIPTION
**Pull request by fivetran-sheringuyen**
Pull Request
**Are you a current Fivetran customer?** 

Fivetran employee

**What change(s) does this PR introduce?** 

Removing keywords join to avoid fanout and rolled back utm logic to Microsoft Ads v0.4.0 implementation.

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [#16 ]
- [ ] No 

**How did you test the PR changes?** 

 

- [x] CircleCi  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Comment by fivetran-sheringuyen** [comment]
To do after PR review:
- [ ] Re-run docs

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# dbt_microsoft_ads v0.5.1
+
+## 🪳Bugfix🪳
+[PR #17](https://github.com/fivetran/dbt_microsoft_ads/pull/17) incorporates the below bugfixes:
+- In v0.5.0, including a join on `keyword_performance_daily_report` caused an unintentional fanout in the `microsoft_ads__url_report` model. We have removed this join and rolled back to the previous logic, which is to use the following logic to extract `utm_term` (Microsoft Ads v0.4.0):
+  - `{{ dbt_utils.get_url_parameter('fields.final_url', 'utm_term') }} as utm_term`
+
+## Contributors placeholder
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:8]: Are there some contributors you want to add here?

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM with just one small comment on the CHANGELOG for the contributors.

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-sheringuyen) [reply CHANGELOG.md:8]: Oh yes, this was a reminder in case the customer on the dbt slack thread responded. But it looks like I could remove this now!

**Pull request merged**